gensrc/thrift/FrontendService.thrift

```diff
@@ -531,6 +531,22 @@ struct TLoadTxnRollbackResult {
     1: required Status.TStatus status
 }
 
+struct TRLTaskCommitRequest {
```

Comment:
I think you should put this request to `TLoadTxnCommitRequest` as an optional field. Then you needn't define a new RPC interface. In addition, if you add new type of RLTask, you just add new option field.

Reply:
For me, all of type of RLTask will commit task by `This method`.  The new option field is in `TRLTaskCommitRequest` not in `TLoadTxnCommitRequest`. Txn just is  a property in RLTask instead of RLTask is a property in Txn.

Reply:
We only commit transaction, task can be finished through callback after transaction commit process.